.travis.yml

```diff
@@ -105,9 +105,16 @@ before_install:
   - bash "${CI_SCRIPTS}/run-epics-iocs-on-procserv.sh"
 
 install:
-  - python -mpip install -Ur requirements.txt
-  - python -mpip install -Ur test-requirements.txt
+  # use a virtual environment  #- python -mpip install -Ur requirements.txt
+  #- python -mpip install -Ur test-requirements.txt
+
+  - python -m venv ophyd-venv
+  - source ophyd-venv/bin/activate
+  - python -mpip install --upgrade pip
```

Comment:
Can we have a blank space between `-m` and `pip` to make it more readable here and 2 lines below?

Reply:
Yes.

Reply:
Also, Travis already uses a virtual environment so I was just making trouble with that.